hls4ml/backends/fpga/passes/repack_stream.py

```diff
@@ -59,6 +59,8 @@ def transform(self, model, node):
 
         # Insert new Repack node instead of Reshape
         repack_layer = model.make_node(Repack, 'repack_' + node.name, attrs, node.inputs.copy())
+        # As result_t attribute is not honored by type conversion, set it manually here
+        repack_layer.attributes[repack_layer.name].type = node.attributes[node.name].type
```

Comment:
(I think this is probably good since it would be more confusing to have multiple copies of a named type, so I think I will resolve it soon unless someone objects in the meantime.)